include/nuttx/can.h

```diff
@@ -1,8 +1,8 @@
-/************************************************************************************
+/****************************************************************************
  * include/nuttx/can/can.h
  *
- *   Copyright (C) 2008, 2009, 2011-2012, 2015-2017, 2019 Gregory Nutt. All rights
- *     reserved.
+ *   Copyright (C) 2008, 2009, 2011-2012, 2015-2017, 2019 Gregory Nutt.
```

Comment:
Hi @jerpelea Greg already submitted this ICLA to convert source code from BSD to Apache. So, should be good to include this modifications when submitting these fixes.

Reply:
I will handle the license in future commits